make_mock_json.py

```diff
@@ -0,0 +1,401 @@
+
+    def create_mock_imagestream(self):
+        print("look in {}".format(self.capture_dir))
+        #imagestream_repo =  "quay.io/prometheus/prometheus"
+        imagestream_server = "registry-proxy-stage.engineering.redhat.com"
+        imagestream_file = "/prometheus/prometheus"
+        imagestream_repo = imagestream_server + imagestream_file
+        try:
+            imagestream = self.osbs.create_image_stream(TEST_IMAGESTREAM, imagestream_repo)
+        except:
+            print("imagstream {} already exists.".format(TEST_IMAGESTREAM))
+            print("run `oc delete imagestream {}` and try again".format(TEST_IMAGESTREAM))
+            exit(0)
+        tags = ['latest', 'master', 'v2.13.0']
+        #tags = ['latest', 'master', '7.2.username-66', '7.2.username-67']
+        self.osbs.import_image_tags(TEST_IMAGESTREAM, tags, imagestream_repo)
+        for tag in tags:
+            self.osbs.ensure_image_stream_tag(imagestream.json(), tag)
+
+        imagestreamimport_fname = "post-namespaces_osbs-stage_imagestreamimports_-000.json"
+        imagestreamimport_path = "/".join([self.capture_dir, imagestreamimport_fname])
+        imagestreamimport_data = []
+        imagestream_data = []
+        with open(imagestreamimport_path, "r") as infile:
+            imagestreamimport_data = json.load(infile)
+
+        imagestreamimport_data["spec"] = {
+            "images": [
+                {
+                    "to": {
+                        "name": "latest"
+                    },
+                    "referencePolicy": {
+                        "type": "Source"
+                    },
+                    "from": {
+                        "kind": "DockerImage",
+                        "name": "example.com/lucarval/rsyslog:latest"
+                    },
+                    "importPolicy": {
+                        "insecure": True
+                    }
+                }
+            ],
+            "import": True
+        }
+
+        self.osbs.get_image_stream(TEST_IMAGESTREAM)
+        imagestream_fname = "get-namespaces_osbs-stage_imagestreams_test_imagestream-001.json"
+        imagestream_path = "/".join([self.capture_dir, imagestream_fname])
+        with open(imagestream_path, "r") as infile:
+            imagestream_data = json.load(infile)
+
+        for i in range(0, 3):
+            tag_line = "7.2.username-{}".format(66 + i)
+            tag_name = "example.com:8888/username/rhel7:" + tag_line
+            spec_tag = imagestream_data["spec"]["tags"][i]
+            spec_tag["from"]["name"] = tag_name
+            spec_tag["name"] = tag_line
+            status_tag = imagestream_data["status"]["tags"][i]
+            status_tag["tag"] = tag_line
+            import_status = imagestreamimport_data["status"]
+            import_status["images"][i]["tag"] = tag_line
+            import_status["import"]["spec"]["tags"][i]["name"] = tag_line
+            import_status["import"]["status"]["tags"][i]["tag"] = tag_line
```

Comment:
Is this just to de-identify it?

Reply:
The tests expect the tags to be ['7.2.username-66', '7.2.username-67', '7.2.username-68'], but import_image_tags() won't work with those tags.  And we can't change the tags in the tests, because the 1.0.4 tests use the old tags.

Reply:
Ah, got it.